code/game/atoms.dm

```diff
@@ -49,6 +49,9 @@
 
 	var/datum/wires/wires = null
 
+	///Used for changing icon states for different base sprites.
+	var/base_icon_state
```

Comment:
Not relevant for this PR directly, but worth noting that there are several other vars that this can replace, that are just named slightly differently. i.e. base_gun_icon for guns etc.